openedx/core/djangoapps/user_authn/views/tests/test_reset_password.py

```diff
@@ -284,7 +283,7 @@ def test_reset_password_email(self, body_type, expected_output):
 
         body = bodies[body_type]
 
-        if django.VERSION >= (3, 0) and body_type == 'html':
+        if body_type == 'html':
             expected_output = "You&#x27;re receiving this e-mail because you requested a password reset"
```

Comment:
Shouldn't the `ddt.data()` parameter above just be updated to match this instead?  I think this whole condition was only added because the HTML entity used in the message changed between Django versions.